deploy/charts/cert-manager/templates/deployment.yaml

```diff
@@ -111,9 +111,7 @@ spec:
           - --leader-election-retry-period={{ .retryPeriod }}
           {{- end }}
           {{- end }}
-          {{- with .Values.acmesolver.image }}
-          - --acme-http01-solver-image={{- if .registry -}}{{ .registry }}/{{- end -}}{{ .repository }}{{- if (.digest) -}} @{{ .digest }}{{- else -}}:{{ default $.Chart.AppVersion .tag }} {{- end -}}
-          {{- end }}
+          - --acme-http01-solver-image="{{ template "image" (tuple .Values.acmesolver.image $.Chart.AppVersion .Values.global) }}
```

Comment:
This is notable, everyone else was using this cute `{{ template "image" (tuple ...) }}`